src/olympia/api/serializers.py

```diff
@@ -40,9 +41,16 @@ def get_fields(self):
                 fields[key] = ESTranslationSerializerField(source=field.source)
         return fields
 
-    def to_native(self, data):
+    def to_internal_value(self, data):
```

Comment:
Is that needed ? serializers inheriting from `BaseESSerializer` should never be used for incoming data, so it should not need to convert representations back to internal values.